src/main/java/net/minecraftforge/client/player/inventory/tabs/GuiTab.java

```diff
@@ -56,9 +58,20 @@
     private Class<? extends GuiContainer> parentContainer;
     private Class<? extends GuiContainer> targetGui;
 
-    public static final GuiTab VANILLA_INVENTORY_TAB = new GuiTab("default_inventory", new ItemStack(Items.SKULL, 1, 3))
+    public static final GuiTab VANILLA_INVENTORY_TAB = new GuiTab("default_inventory")
     {
         @Override
+        public ItemStack getIconStack()
+        {
+            ItemStack skull = new ItemStack(Items.SKULL, 1, 3);
+            skull.setTagCompound(new NBTTagCompound());
+            NBTTagCompound nbttagcompound = new NBTTagCompound();
+            NBTUtil.writeGameProfile(nbttagcompound, Minecraft.getMinecraft().getSession().getProfile());
```

Comment:
Currently the player skull (with the Player's skin) is only visible in Single Player.
But when joining a server the skull gets replaced with the default (Steve/Alex Skin).

Reply:
How about the skull of the client player?

Reply:
why's it have to be the player's head, can't it just be the default steve head?
it's cool and all but if there's problems with it it's not really worth dealing with

Reply:
Boy, it is not that hard... You can get a skull of a certain player fairly easily.

Reply:
I mean like, the "cool" look will be when the player plays on single player. but when being on a server the default skin (steve/alex) will display. I guess its not that important ?

Also liach, would love to know how I could do that :)